bin/local-dev/wordpress/Dockerfile

```diff
@@ -37,9 +38,8 @@ RUN buildDeps='xz-utils' \
     A48C2BEE680E841632CD4E44F07496B3EB3C1762 \
     B9E2F5981AA6E0CD28160D9FF13993A75599653C \
   ; do \
-    gpg --batch --keyserver hkp://p80.pool.sks-keyservers.net:80 --recv-keys "$key" || \
-    gpg --batch --keyserver hkp://ipv4.pool.sks-keyservers.net --recv-keys "$key" || \
-    gpg --batch --keyserver hkp://pgp.mit.edu:80 --recv-keys "$key" ; \
+    gpg --batch --keyserver hkps://keyserver.ubuntu.com --recv-keys "$key" || \
```

Comment:
Change to working key servers as the old ones are not resolving anymore per https://github.com/nodejs/docker-node/issues/1500